src/translib/transformer/sw_portchannel.go

```diff
@@ -486,6 +485,13 @@ func deleteLagIntfAndMembers(inParams *XfmrParams, lagName *string) error {
         return errors.New(errStr)
     }
 
+    /* Restrict deletion if iface configured as member-port of any Vlan */
+    err = validateIntfAssociatedWithVlan(inParams.d, lagName)
+    if err != nil {
+        errStr := "Interface configured as member-port of Vlan"
```

Comment:
Better to have error message like, VLAN: 10 configuration exists on Interface: Ethernet24.